common/types/shared.go

```diff
@@ -8521,30 +8617,30 @@ func (e TimeoutType) String() string {
 	w := int32(e)
 	switch w {
 	case 0:
-		return "START_TO_CLOSE"
+		return "StartToClose"
 	case 1:
-		return "SCHEDULE_TO_START"
+		return "ScheduleToStart"
 	case 2:
-		return "SCHEDULE_TO_CLOSE"
+		return "ScheduleToClose"
 	case 3:
-		return "HEARTBEAT"
+		return "Heartbeat"
```

Comment:
We can't simply change this representation, as it will be used as the timeout error reason. User needs to type the exact same string when specifying non-retryable errors. See https://github.com/uber/cadence/blob/master/service/history/execution/timer_sequence.go#L440